plugins/node/instrumentation-undici/src/types.ts

```diff
@@ -41,6 +41,7 @@ export interface UndiciRequest {
 export interface UndiciResponse {
   headers: Buffer[];
   statusCode: number;
+  statusText: string;
```

Comment:
What is the motivation to add this field? #2179 mentions status code and not text

Reply:
The response object contains this property so why not expose it in the types? 

Users could spread the response into a new object not knowing it has this property:
```ts
const data = {
  ...someOtherObj,
  ...response,
}
```

Reply:
I'd say it's fine to have this field if it's available in all supported versions. If not you can make it optional.

Reply:
As far as I can tell, this has existed as long as the diagnostic channel has had type defs:
https://github.com/nodejs/undici/blame/9bc9b44a24d7869f568103b4c7cf428755601157/types/diagnostics-channel.d.ts#L16

Reply:
Ok, I thought it was related to an issue.
Not sure when it is useful, but don't mind having it in the hook 👍